### PR TITLE
update quoting.md

### PR DESCRIPTION
**Pull request by mirnawong1**
Amended quoting defaults based on this thread here: https://dbt-labs.slack.com/archives/CFQ823PMH/p1641378335133800

and documents here: https://docs.getdbt.com/reference/project-configs/quoting AND https://docs.getdbt.com/reference/resource-properties/quoting#default

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [x] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/reference/resource-properties/quoting.md [hunk 1]
```diff
@@ -37,8 +37,15 @@ Note that for BigQuery quoting configuration, `database` and `schema` should be
 
 
 ## Default
+The default values vary by database. 
 
-By default, dbt will _not_ quote the database, schema, or identifier.
+For most adapters, quoting is set to _true_ by default.
+
+This is because creating relations with quoted identifiers does not inhibit the ability to select from them, and quoting allows you to use reserved words as object names (though that should probably be avoided)
```
Comment (jtcohen6) [inline website/docs/reference/resource-properties/quoting.md]:
```suggestion
Why? It's equally easy to select from relations with quoted or unquoted identifiers. Quoting allows you to use reserved words and special characters in those identifiers, though we recommend avoiding this whenever possible.
```

File: website/docs/reference/resource-properties/quoting.md [hunk 2]
```diff
@@ -37,8 +37,15 @@ Note that for BigQuery quoting configuration, `database` and `schema` should be
...
+
+On Snowflake, quoting is set to _false_ by default.
+
+This is because creating relations with quoted identifiers inhibits your ability to select from the them.
```
Comment (jtcohen6) [inline website/docs/reference/resource-properties/quoting.md]:
```suggestion
Creating relations with quoted identifiers also makes those identifiers case sensitive. It's much more difficult to select from them. You can re-enable quoting for relations identifiers that are case sensitive, reserved words, or contain special characters, but we recommend you avoid this as much as possible.
```
```

**Review by jtcohen6 — COMMENTED** [review]
Thanks for making the change @mirnawong1!

I left a couple (imperfect) recommendations to make the language clearer. For any language adjustments we do make here, let's also adjust language in https://docs.getdbt.com/reference/project-configs/quoting#default

File: website/docs/reference/resource-properties/quoting.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/reference/resource-properties/quoting.md]: gotcha, thanks and have submitted those changes as well!

Comment (mirnawong1) [reply website/docs/reference/resource-properties/quoting.md]: pr submitted here: https://github.com/dbt-labs/docs.getdbt.com/pull/1005 thank you @jtcohen6 !

**Comment by coapacetic** [comment]
👋 @mirnawong1 I'm doing some GitHub clean up, did you handle everything that was covered here in a different branch?  Feel free to close if that's the case.  Otherwise, happy to dive into a deeper review.

**Comment by mirnawong1** [comment]
hi @coapacetic ! yes i believe i submitted another pr [here](https://github.com/dbt-labs/docs.getdbt.com/pull/1005)

i think this one is ok as well?

**Comment by coapacetic** [comment]
oh yea! @mirnawong1 looks like you covered all these changes elsewhere. You can click `close with comment` so that this isn't a stale PR sitting around for the future.

**Review by coapacetic — APPROVED** [review]
LGTM - but I think this was merged elsewhere

**Pull request merged**